packages/aws-cdk/lib/commands/init/init.ts

```diff
@@ -270,32 +490,33 @@ interface ProjectInfo {
 }
 
 export async function availableInitTemplates(): Promise<InitTemplate[]> {
-  return new Promise(async (resolve) => {
-    try {
-      const templatesDir = path.join(cliRootDir(), 'lib', 'init-templates');
-      const templateNames = await listDirectory(templatesDir);
-      const templates = new Array<InitTemplate>();
-      for (const templateName of templateNames) {
-        templates.push(await InitTemplate.fromName(templatesDir, templateName));
-      }
-      resolve(templates);
-    } catch {
-      resolve([]);
+  try {
+    const templatesDir = path.join(cliRootDir(), 'lib', 'init-templates');
+    const templateNames = await listDirectory(templatesDir);
+    const templatePromises = templateNames.map(templateName =>
+      InitTemplate.fromName(templatesDir, templateName),
+    );
+    // eslint-disable-next-line @cdklabs/promiseall-no-unbounded-parallelism
```

Comment:
provide a justification for this eslint disabled rule, like we have elsewhere when we disable this